src/core/Tag/index.stories.tsx

```diff
@@ -102,179 +97,116 @@ export default {
 
 const Template: Story = (args) => <Demo {...args} />;
 
-export const Primary = Template.bind({});
-
-Primary.args = {
-  label: "Label",
-  sdsStyle: "square",
-  sdsType: "primary",
-};
-
-export const RoundedPrimary = Template.bind({});
-
-RoundedPrimary.args = {
-  label: "Label",
-  sdsStyle: "rounded",
-  sdsType: "primary",
-};
-
-export const PrimaryWithIcon = Template.bind({});
+export const Default = Template.bind({});
 
-PrimaryWithIcon.args = {
-  icon: <Icon sdsSize="l" sdsIcon="checkCircle" sdsType="button" />,
+Default.args = {
   label: "Label",
   sdsStyle: "square",
   sdsType: "primary",
 };
 
-export const RoundedPrimaryWithIcon = Template.bind({});
-
-RoundedPrimaryWithIcon.args = {
-  icon: <Icon sdsSize="l" sdsIcon="checkCircle" sdsType="button" />,
-  label: "Label",
-  sdsStyle: "rounded",
-  sdsType: "primary",
-};
-
-export const Secondary = Template.bind({});
-
-Secondary.args = {
-  label: "Label",
-  sdsStyle: "square",
-  sdsType: "secondary",
-};
-
-export const RoundedSecondary = Template.bind({});
-
-RoundedSecondary.args = {
-  label: "Label",
-  sdsStyle: "rounded",
-  sdsType: "secondary",
-};
-
-export const SecondaryWithIcon = Template.bind({});
-
-SecondaryWithIcon.args = {
-  icon: <Icon sdsSize="l" sdsIcon="checkCircle" sdsType="button" />,
-  label: "Label",
-  sdsStyle: "square",
-  sdsType: "secondary",
-};
-
-export const RoundedSecondaryWithIcon = Template.bind({});
-
-RoundedSecondaryWithIcon.args = {
-  icon: <Icon sdsSize="l" sdsIcon="checkCircle" sdsType="button" />,
-  label: "Label",
-  sdsStyle: "rounded",
-  sdsType: "secondary",
-};
-
-export const DoubleColorWithoutIcon = Template.bind({});
-
-DoubleColorWithoutIcon.args = {
-  color: ["brown", "lightsalmon"],
-  label: "Label",
-  sdsStyle: "rounded",
-  sdsType: "secondary",
-};
-
-export const TripleColorWithIcon = Template.bind({});
-
-TripleColorWithIcon.args = {
-  color: ["brown", "lightsalmon", "crimson"],
-  icon: <Icon sdsSize="l" sdsIcon="checkCircle" sdsType="button" />,
-  label: "Label",
-  sdsStyle: "rounded",
-  sdsType: "secondary",
-};
-
 /*
- * Live Preview for Rounded Tag
+ * Live Preview
  */
 
 const livePreviewStyles = {
-  columnGap: "24px",
-  display: "flex",
+  display: "inline-grid",
+  gridColumnGap: 24,
+  gridRowGap: 24,
+  gridTemplateColumns: "repeat(3, auto)",
+  gridTemplateRows: "repeat(2, auto)",
 };
 
-const TagLivePreviewDemo = (props: Args): JSX.Element => {
-  const { color, icon, label, sdsStyle } = props;
+const LivePreviewDemo = (props: Args): JSX.Element => {
```

Comment:
Ahh this is so much cleaner! Thanks @masoudmanson  🙏

Reply:
Exactly, I followed CSF format for writing stories 😎